core/src/main/scalajvm/sttp/client3/quick.scala

```diff
@@ -1,5 +1,6 @@
 package sttp.client3
 
 object quick extends SttpApi {
-  lazy val backend: SttpBackend[Identity, Any] = HttpURLConnectionBackend()
+  lazy val backend: SttpBackend[Identity, Any] = HttpClientSyncBackend()
+  lazy val urlConnectionBackend: SttpBackend[Identity, Any] = HttpURLConnectionBackend()
```

Comment:
let's remove this and keep just `backend`

Reply:
but which one should be the exact backend: URL or HttpClient ?

Reply:
we're keeping `HttpClient` of course :)